tensilelite/rocisa/rocisa/src/helper.cpp

```diff
@@ -29,8 +29,9 @@
 
 #ifdef WIN32
 
-#include <dbghelp.h>
 #include <windows.h>
+// windows.h must be loaded before other windows headers.
+#include <dbghelp.h>
```

Comment:
Please wrap 
// clang-format off/on around these headers

Reply:
Or just simply add an empty line

Reply:
I usually just use header groups like this as clang-format will not sort across different groups of headers (blank lines or comments delineate groups).